### DISCUSSION TITLE
making-my-first-contribution-on-sourceforge-using-mercurial/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#42] [Announcements]
# making-my-first-contribution-on-sourceforge-using-mercurial/

In this post I describe my experience of making my first contribution to a project on SourceForge, using the Mercurial version control system.

https://andrewlock.net/making-my-first-contribution-on-sourceforge-using-mercurial/

**Comment by kberg** [comment]
This was exactly the reference I needed. Thank you!

[a comment by an automated account is not shown]
